test/functional/tests/stats/test_display_statistics.py

```diff
@@ -0,0 +1,420 @@
+                    f"core device is {cores_stats[j].config_stats.core_dev}, "
+                    f"should be {cores[i][j].core_device.system_path}\n")
+            if cores_stats[j].config_stats.core_size.value != cores[i][j].size.value:
+                failed_stats += (
+                    f"For exported object {cores[i][j].system_path} "
+                    f"core size is {cores_stats[j].config_stats.core_size.value}, "
+                    f"should be {cores[i][j].size.value}\n"
+                )
+            if cores_stats[j].config_stats.dirty_for.total_seconds() < 0:
+                failed_stats += (
+                    f"For exported object {cores[i][j].system_path} value of "
+                    f"'Dirty for' is {cores_stats[j].config_stats.dirty_for}, "
+                    f"should be greater or equal 0\n")
+            if (
+                CoreStatus[cores_stats[j].config_stats.status.lower()]
+                != cores[i][j].get_status()
+            ):
+                failed_stats += (
+                    f"For exported object {cores[i][j].system_path} cache "
+                    f"status is {cores_stats[j].config_stats.status}, should be "
+                    f"{str(cores[i][j].get_status()).split('.')[1].capitalize()}\n")
+            if cores_stats[j].config_stats.seq_cutoff_policy is None:
+                failed_stats += (
+                    f"For exported object {cores[i][j].system_path} value of "
+                    f"Sequential cut-off policy should not be empty\n")
+            if cores_stats[j].config_stats.seq_cutoff_threshold.value <= 0:
+                failed_stats += (
+                    f"For exported object {cores[i][j].system_path} value of "
+                    f"Sequential cut-off threshold should be greater then 0\n")
+
+    if failed_stats:
+        TestRun.LOGGER.error(
+            f"There are some inconsistencies in core "
+            f"configuration statistics:\n{failed_stats}")
+
+
+def validate_statistics_flat(device, stats, stat_filter, per_core: bool):
+    failed_stats = ""
+    if stat_filter == StatsFilter.usage:
+        current_stats = usage_stats
+    if stat_filter == StatsFilter.blk:
+        current_stats = block_stats_core if per_core else block_stats_cache
+    if stat_filter == StatsFilter.req:
+        current_stats = request_stats
+    if stat_filter == StatsFilter.err:
+        current_stats = error_stats
+
+    for stat_name in current_stats:
+        if stat_name not in stats.keys():
+            failed_stats += (
+                f"For core device {device.system_path} value for "
+                f"{stat_name} not displayed in output\n"
+            ) if per_core else (
+                f"For cache nr {device.cache_id} value for "
+                f"{stat_name} not displayed in output\n")
+        else:
+            del stats[stat_name]
+    if len(stats.keys()):
+        failed_stats += (
+            f"For core device {device.system_path} additional "
```

Comment:
with values this sentence order sounds better, but here I would change it to:
Additional statistics found for core ... :

Reply:
Done